packages/node/Dockerfile

```diff
@@ -23,21 +23,16 @@ FROM node:18-alpine
 COPY --from=builder /app/packages/node/app.tgz /app.tgz
 
 # Install production dependencies
-RUN apk add --no-cache tini curl git &&  \
-    apk add --no-cache python3 make g++ &&  \
-    npm install -g node-gyp
-
-# Install the app
-RUN npm i -g app.tgz
-
-# Clean up unused deps
-RUN rm /app.tgz && \
+RUN apk add --no-cache tini curl git && \
+    tar -xzvf /app.tgz --strip 1 && \
+    rm /app.tgz && \
+    yarn install --production && \
     yarn cache clean && \
     rm -rf /root/.npm /root/.cache
 
 # Make the user not ROOT
 USER 1000
 
 # Set Entry point and command
-ENTRYPOINT ["/sbin/tini", "--", "/usr/local/lib/node_modules/@subql/node-algorand/bin/run"]
+ENTRYPOINT ["/sbin/tini", "--", "/bin/run"]
```

Comment:
I thought we were going to keep this as it was, but pull over the CI release/prerelease changes